go/libraries/doltcore/sqle/indexed_dolt_table.go

```diff
@@ -152,6 +152,42 @@ func (t *WritableIndexedDoltTable) NumRows(ctx *sql.Context) (uint64, error) {
 	return m.Len(), nil
 }
 
+func (t *WritableIndexedDoltTable) DataLength(ctx *sql.Context) (uint64, error) {
```

Comment:
`WritableIndexedDoltTable` embeds a `DoltTable`, so you shouldn't need to reimplement this